dnd5esheets/front/src/routes/character/[slug].tsx

```diff
@@ -1,157 +1,21 @@
-import { ComponentProps } from "solid-js";
+import { ComponentProps, For } from "solid-js";
```

Comment:
🌱 These are both unused (and there are more in the codebase). We should setup some linters/checks to look for these small violations.